addons/event/static/src/client_action/event_barcode.scss

```diff
@@ -24,6 +24,30 @@
         max-width: 200px;
         max-height: 100px;
     }
+    .o_barcode_mobile_container {
+        margin-top: 75px;
+        margin-bottom: -35px;
+        img {
+            height: 185px;
+            width: 275px;
+        }
+        // In order to have the o_mobile_barcode button on both the image and the label,
+        // We use negative margin at the bottom and 0 opacity (since not needed in the view)
+        .o_mobile_barcode {
+            opacity: 0;
+            height: 220px;
+            width: 275px;
+            bottom: -35px;
+        }
+        .o_barcode_laser {
+            height: 3px;
+            width: 125%;
+            left: -12.5%;
```

Comment:
did not see this, same comment, if we can be more explicit or use relative positions, would probably be better.

Reply:
(both width and left actually)